examples/messages.ts

```diff
@@ -0,0 +1,36 @@
+import { Client, Message } from "../mod.ts";
+
+const client = new Client({
+  id: Deno.env.get("CLIENT_ID")!,
+  secret: Deno.env.get("CLIENT_SECRET")!,
+  scopes: ["rpc", "messages.read"],
+});
+
+(async () => {
+  for await (const event of client) {
```

Comment:
wrapping inside async is not required with TLA

Reply:
that for await loop will block forever until client is disconnected